src/sage/categories/finite_lattice_posets.py

```diff
@@ -37,6 +41,20 @@ class FiniteLatticePosets(CategoryWithAxiom):
         True
         sage: TestSuite(C).run()
     """
+    @cached_method
+    def extra_super_categories(self):
+        r"""
+        Return a list of the (immediate) super categories of
+        ``self``, as per :meth:`Category.super_categories`.
+
+        EXAMPLES::
+
+            sage: FiniteLatticePosets().super_categories()
+            [Category of lattice posets,
+             Category of finite posets,
+             Category of bounded posets]
+        """
+        return [Posets().Bounded()]
```

Comment:
```suggestion
        return [Lattices().Bounded()]
```
If you make this change, does the "bounded" get removed from the category printing? This isn't a big issue though, but since bounded is implied by finite, in principle, it should not be printed according to the category framework. It's possible to have this work, you would need another trivial category `Bounded` in `Lattices`.

Reply:
I have made your proposed changes, but the "bounded" does not disappear. I will try adding a "Bounded" subcategory in lattices.